runtime/doc/msgpack_rpc.txt

```diff
@@ -264,10 +264,13 @@ a dictionary with these (optional) keys:
 				colors.
 				Set to false to use terminal color codes (at
 				most 256 different colors).
-	`popupmenu_external`	Instead of drawing the completion popupmenu on
-				the grid, Nvim will send higher-level events to
-				the ui and let it draw the popupmenu.
-				Defaults to false.
+	`ui_ext`		String array of "externalized" widgets.
+				Widgets in this list here will not be drawn by
```

Comment:
Un-document `popupmenu_external` but continue to support it. We can try removing it on master after 0.2 and see if it breaks anything.